testutils/api/deviceconnect.py

```diff
@@ -15,3 +15,4 @@
 URL_MGMT_DEVICE = "/devices/{id}"
 URL_MGMT_CONNECT = "/devices/{id}/connect"
 URL_MGMT_PLAYBACK = "/sessions/{session_id}/playback"
+URL_MGMT_FDOWNLOAD = "/devices/{id}/download"
```

Comment:
Shall we test the /upload end-point, as well?

Reply:
the idea was to test each feature very lightly, but sure, it's a cheap addition so I'll add it.